File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/firebase/dynamiclinks/DynamicLinksActivity.java
package com.jdroid.android.sample.ui.firebase.dynamiclinks;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class DynamicLinksActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return DynamicLinksFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/firebase/dynamiclinks/DynamicLinksActivity.kt
package com.jdroid.android.sample.ui.firebase.dynamiclinks

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class DynamicLinksActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return DynamicLinksFragment::class.java
    }
}
